examples/06-distributed-post/01-distributed_workflows_on_remote.py

```diff
@@ -0,0 +1,74 @@
+"""
+.. _ref_distributed_workflows_on_remote:
+
+Connect operator chains on different processes implicitly
```

Comment:
Could you please rename this one to: "Create custom workflow on distributed processes."